### DISCUSSION TITLE
How to enable code covered on WSL

### DISCUSSION THREAD

**Discussion by kicaj29** [coverlet-coverage/coverlet#1702] [Q&A]
Hi 

When I run dotnet test on Windows I have code coverage in my output. When I do it on WSL (Ubuntu) then all tests are passing but there is no code coverage report.
I am using such command (also to see more logs):

```
dotnet test --filter "FullyQualifiedName=FullyQualifiedName=Namespace.ClassName.MethodName" --no-build -c Debug -v:d /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:CoverletOutput=./coverage/ 
```

Why code coverage report is not generated?

**Comment by Bertk** [comment]
Please check your .NET versions available with WSL. coverlet V6.0.4 requires .NET 6.0 runtime.

see https://documentation.ubuntu.com/ubuntu-for-developers/reference/availability/dotnet/

```shell
me@LAPTOP:~$ dotnet --list-sdks
8.0.416 [/usr/share/dotnet/sdk]
9.0.203 [/usr/share/dotnet/sdk]
```

You should consider using DevContainer (see PR [Add DevContainer using .NET 10](https://github.com/coverlet-coverage/coverlet/pull/1783)).

**Comment by Bertk** [comment]
We added a [devcontainer](https://github.com/coverlet-coverage/coverlet/tree/master/.devcontainer) which can be used for a linux terminal session.
